packages/components/src/custom-select-control-v2/styles.ts

```diff
@@ -108,7 +108,14 @@ export const SelectItem = styled( Ariakit.SelectItem )`
 	justify-content: space-between;
 	padding: ${ ITEM_PADDING };
 	font-size: ${ CONFIG.fontSize };
-	line-height: 2.15rem; // TODO: Remove this in default but keep for back-compat in legacy
```

Comment:
To match 28px, the line height value should have been `2.15` (without the `rem` unit). I felt like a more explicit `calc` syntax would help to understand the reason for an arbitrary number.

Reply:
It's curious where the 2.15 came from originally...

Reply:
It's the ratio of the line height (28px) to the font size (13px)

Also, since `28 / 13 = 2.15384615`, another consequence of using `2.15` is that the resulting line height wouldn't be exactly 28px (as per design), but slightly less because of the lower decimal precision.

In short: I prefer to let the browser (or sass) do the calculation, and if not possible, to state the `line-height` directly.

Reply:
> Also, since 28 / 13 = 2.15384615, another consequence of using 2.15 is that the resulting line height wouldn't be exactly 28px (as per design), but slightly less because of the lower decimal precision.

Which also makes me question if 28/13 was the intent. But yes, having a clear calc there makes things much clearer.